Lib/wsgiref/types.py

```diff
@@ -0,0 +1,53 @@
+"""WSGI-related types for static type checking"""
```

Comment:
Might be good to add a note to the module docstring that any changes contributors want to make to this file will need to be made to typeshed simultaneously if they're to have any meaning to static type checkers

Reply:
I'm not sure we should mention typeshed explicitly for a few reasons:

* I don't want to burden CPython contributors to need to contribute to typeshed.
* In typeshed we are pretty good in picking up changes, and for CPython we have quite some time to make these changes.
* While this is currently not the case, type checkers could in theory use the implementation module to type check instead of typeshed. In fact, I have an idea to optionally direct type checkers to the stdlib in cases like this, so it's possible that this comment would get outdated during the life time of Python 3.11.